src/main/java/org/jabref/gui/Base.css

```diff
@@ -1308,6 +1308,10 @@ We want to have a look that matches our icons in the tool-bar */
     -fx-text-fill: -jr-warn;
 }
 
+.warning-label {
```

Comment:
Why couldn't you use the class `.warning-message` (instead of introducing a new class) with the color `-jr-warn`? (See line 76)

JabRef tries to have consistent UI colors.

Reply:
Yes, actually I have night light on on my laptop, so couldn't see the orange warning too well. So I stuck to red.
Then when I realized reason was the night light was the reason, removed it. Sorry.